articles/machines/qc-trace-simulator/index.md

```diff
@@ -81,9 +81,10 @@ namespace Quantum.MyProgram
 }
 ```
 
-Note that if there is at least one measurement not annotated using `AssertProb`
-or `ForceMeasure` the simulator will throw `UnconstraintMeasurementException`
-from the `Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime` namespace. See the API documentation on [UnconstraintMeasurementException](https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.unconstraintmeasurementexception) for more details.
+Note that if there is at least one measurement not annotated using `AssertProb`,
+the simulator will throw `UnconstrainedMeasurementException`
+from the `Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators` namespace. See the API documentation on 
+[UnconstrainedMeasurementException](https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.unconstrainedmeasurementexception) for more details.
```

Comment:
I think it be better long-term to make this an xref to the .NET API Browser than a URL, as that should persist through changes in how the .NET API Browser is laid out. 
```suggestion
[UnconstrainedMeasurementException](xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.UnconstrainedMeasurementException) for more details.
```